deps/debugging-tools.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# vagrant-cookbook
+# Copyright (C) 2015 Pelagicore AB
```

Comment:
I actually saw this and thought the same thing, and then forgot about it. Logically, yes. Do you agree?